static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -86,13 +93,25 @@ function importSingleProfile(
   frameIndex: ReturnType<typeof createFrameIndex>
 ): Profile {
   if (isEventedProfile(profile)) {
-    return EventedProfile.FromProfile(profile, frameIndex);
+    return wrapWithSpan(() => EventedProfile.FromProfile(profile, frameIndex), {
+      op: 'profile.import',
+      description: 'evented',
+    });
   }
   if (isSampledProfile(profile)) {
-    return SampledProfile.FromProfile(profile, frameIndex);
+    return wrapWithSpan(() => SampledProfile.FromProfile(profile, frameIndex), {
+      op: 'profile.import',
+      description: 'sampled',
+    });
   }
   if (isJSProfile(profile)) {
-    return JSSelfProfile.FromProfile(profile, createFrameIndex(profile.frames));
+    return wrapWithSpan(
+      () => JSSelfProfile.FromProfile(profile, createFrameIndex(profile.frames)),
+      {
+        op: 'profile.import',
+        description: 'js',
```

Comment:
We'll have other profiles like v8 etc in the future
```suggestion
        description: 'js-self-profile',
```